src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -384,6 +423,29 @@ static void auh_entry_store(TimestampTz auh_time,
   LWLockRelease(auh_entry_array_lock);
 }
 
+static void table_info(const char* id,
+                       const char* name,
+                       uint32_t table_type,
+                       uint32_t relation_type,
+                       const char* namespace_id,
+                       const char* namespace_name,
+                       uint32_t database_type,
+                       const char* pgschema_name,
+                       bool colocated,
+                       const char* parent_table_id) {
+    ybtableInfo tableInfo;
```

Comment:
What is this function doing? 

It seems to return nothing.

Reply:
It was created so that based on what the table ID information is to be collected for, I can populate the global fields later (Similar to the circular buffer), but I can remove it for now.